src/newrelic/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = ( { logger = console } = {} ) => {
+	const licenseKey = process.env.NEW_RELIC_LICENSE_KEY;
+	const noConfig = process.env.NEW_RELIC_NO_CONFIG_FILE;
+	const isProduction = () => process.env.NODE_ENV === 'production';
```

Comment:
I don't think we need the `NODE_ENV` check. We may have a non-production environment hosted on Go as well.